CheatSheets/pandas-cheetcode.md

```diff
@@ -0,0 +1,175 @@
+## Table of Contents
+
+- [Pandas CheatSheet for Developers](#Pandas-cheatsheet-for-developers)
+- [Introduction-What-is-Pandas?](#introduction-what-is-Pandas)
+- [Key and Imports?](#Key-and-Imports)
+- [Importing Data](#Importing-Data)
+- [Exorting Data](#Exporting-Data)
+- [Create Test objects](#Create-Test-objects)
+- [Viewing/Inspecting Data](#Viewing/Inspecting-Data)
+- [Selection](#Selection)
+- [Data cleaning](#Data-cleaning)
+- [Filter, Sort, and Groupby](#Filter-Sort-and-Groupby)
+- [Join/Combine](#Join/Combine)
+- [Statistics](#Statistics)
```

Comment:
> Please, give me some `reference links` so that I can `verify` your every `keyword or command` is `correct or not`.

https://pandas.pydata.org/docs/user_guide/index.html
https://www.w3schools.com/python/pandas/default.asp
actually i have followed the text booked so it will be correct dont worry